wrappers/rust/rust-toolchain

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.70.0"
+channel = "1.72.0"
```

Comment:
Consider removing the whole file

Reply:
or downgrading it to 1.69.0

Reply:
I already removed it so just rebase/merge.